templates/index.html

```diff
@@ -2,31 +2,60 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-<div class="col-md-12">
-  <table class="table table-striped">
-    <thead>
-      <tr>
-        <th scope="col">id</th>
-        <th scope="col">Title</th>
-        <th scope="col">Status</th>
-        <th scope="col">Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      {% for item in items %}
-      <tr>
-        <th scope="row">{{ item['id']}}</th>
-        <td>{{ item['title']}}</td>
-        <td>{{ item['status'] }}</td>
-        <td>
-          <form class="form-inline" action="" method="PUT">
-            <button type="submit" class="btn btn-success" name="CompleteTask"><i class="fa fa-check" aria-hidden="true"></i>&nbspComplete</button>
-            <button type="submit" class="btn btn-danger" name="CompleteTask"><i class="fa fa-trash" aria-hidden="true" disabled></i>&nbspDelete</button>
-          </form>
-        </td>
-      </tr>
+<div class="Form">
+<div class="base">
+  <div class="board">
+    <div class="board-header">
+      <div class="left">
+        <div class="board-header-text">{{ boards[0]['name'] }}</div>
```

Comment:
Similarly to below - this is fine but we don't gain much by fetching the board directly; we could simplify this by just explicitly setting our chosen header text

Reply:
this has now been removed and added to the Navbar, whilst I understand what you are saying, ( to me at least ) it feels nicer to be using the actual board name set by the user on their Trello board